mpt/impl/imap-mailbox/pom.xml

```diff
@@ -47,15 +47,7 @@
     </modules>
 
     <properties>
-        <cassandra.includes>empty</cassandra.includes>
         <hbase.includes>empty</hbase.includes>
```

Comment:
why ?

Reply:
Hbase test do not pass...

Reply:
then we should put @Ignore on the failing tests instead IMO